lang/java/reef-runtime-hdinsight/src/main/java/org/apache/reef/runtime/hdinsight/client/yarnrest/ApplicationState.java

```diff
@@ -18,16 +18,20 @@
  */
 package org.apache.reef.runtime.hdinsight.client.yarnrest;
 
+import com.sun.xml.bind.v2.runtime.reflect.opt.Const;
```

Comment:
What is this needed for? We should avoid importing private sun APIs wherever possible.